google-cloud-logging/src/main/java/com/google/cloud/logging/Instrumentation.java

```diff
@@ -114,7 +114,7 @@ public static WriteOption[] addPartialSuccessOption(WriteOption[] options) {
    *     with 'java'. Will be truncated if longer than 14 characters.
    * @param libraryVersion {string} The version of the logging library to be reported. Will be
    *     truncated if longer than 14 characters.
-   * @returns {LogEntry} The entry with diagnostic instrumentation data.
+   * @return {LogEntry} The entry with diagnostic instrumentation data.
```

Comment:
I think lint test fails on this file and I guess it is related to a comment you changed

Reply:
oops, fixed